context/.eslintrc.yml

```
@@ -10,6 +10,8 @@ extends:
   - plugin:import/errors
   - plugin:import/warnings
 parser: "babel-eslint"
+globals:
+  IMAGES_CDN_ORIGIN: readonly
```

```suggestion
  CDN_URL: readonly
```
- May hold assets besides images someday?
- For the CDN, "origin" means where the assets originally come from: In our case, that's S3, and that's not what you mean here.